code/game/objects/items/devices/radio/headset.dm

```diff
@@ -367,7 +367,7 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	// "puts the antenna down" while the headset is off
 	overlay_speaker_idle = "headset_up"
 	overlay_mic_idle = "headset_up"
-	keyslot = /obj/item/encryptionkey/headset_mining
+	keyslot = /obj/item/encryptionkey/headset_cargo // IRIS EDIT - changed to cargo key
```

Comment:
Just comment it out, the parent already has the keyslot defined as headset_cargo
```suggestion
//	keyslot = /obj/item/encryptionkey/headset_cargo // IRIS REMOVAL - Removed miners having science comms
```